src/hotspot/share/interpreter/oopMapCache.hpp

```diff
@@ -128,11 +129,12 @@ class InterpreterOopMap: ResourceObj {
 
  public:
   InterpreterOopMap();
+  ~InterpreterOopMap();
 
   // Copy the OopMapCacheEntry in parameter "from" into this
   // InterpreterOopMap.  If the _bit_mask[0] in "from" points to
   // allocated space (i.e., the bit mask was to large to hold
-  // in-line), allocate the space from a Resource area.
+  // in-line), allocate the space from the C heap.
   void resource_copy(OopMapCacheEntry* from);
```

Comment:
The name `resource_copy` seems somewhat of a misnomer given it may be C heap. Is it worth changing?

Reply:
I agree, this should probably be copy_from, and rename the parameter src.  Or something like that.

Reply:
I also thought about renaming it but ended up leaving it as is in v1. I changed it to Coleen's suggestion.